src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -0,0 +1,119 @@
+@snippet methodSampleDoc(docConfig)
+    @let ApiName = docConfig.getApiName, \
+            apiName = context.upperCamelToLowerCamel(ApiName), \
+            methodName = docConfig.getMethodName(), \
+            returnType = docConfig.getReturnType(), \
+            initCodeSpec = docConfig.getInitCode()
+        try {
+            ${@apiName} = new {@ApiName}();
+            {@initCode(initCodeSpec)}
+            @if docConfig.isPagedVariant()
+                for ({@methodCallSampleCode(methodName, apiName, initCodeSpec)} as element) {
```

Comment:
@garrettjonesgoogle Yea sorry you are right, the key will be the index for sequential array.

For now the key for PageAccessor will be the request object so it makes sense to have a key/value iteration in the sample code.